docusaurus/website/static/css/custom.css

```diff
@@ -32,6 +32,9 @@
 .headerTitleWithLogo {
   white-space: nowrap;
 }
+.projectLogo {
```

Comment:
```suggestion

.projectLogo {
```

Reply:
@iansu Thanks for the suggestion. I have committed the changes now.